### DISCUSSION TITLE
规则里面有排除某个域名的写法吗

### DISCUSSION THREAD

**Discussion by dongxicc** [v2fly/v2ray-core#2364] [Q&A]
如题,类似下面的规则:
```
{
                "type": "field",
                "domain": [
                    "!baidu.com",
                    "getsite:cn"
                ],
                "outboundTag": "direct"
            }
```
例如只是不想让 baidu 走direct, ! baidu.com 这种支持吗?
确实可以单独加一个规则 baidu.com 让他走 proxy,但是这不符合我的需求,因为后面还有一些细分 针对 sourceIp 的规则

**Comment by mydogshitgold** [comment]
You can use regular expression (starts with `regexp:`), but adding an additional rule may make it clearer. The rules will apply in descending order.
